pkg/commands/info.go

```diff
@@ -74,7 +75,20 @@ func RunInfo(ctx CommandContext, args InfoArgs) error {
 	fmt.Fprintf(ctx.Stdout, "  Images:\t\t%d\n", dashboard.ImagesCount)
 	fmt.Fprintf(ctx.Stdout, "  Snapshots:\t\t%d\n", dashboard.SnapshotsCount)
 	fmt.Fprintf(ctx.Stdout, "  Servers:\t\t%d\n", dashboard.ServersCount)
-	fmt.Fprintf(ctx.Stdout, "  Ips:\t\t\t%d\n", dashboard.IPsCount)
+	fmt.Fprintf(ctx.Stdout, "  Ips:\t\t\t%d\n\n", dashboard.IPsCount)
```

Comment:
can you add the `\n` on a dedicated print to keep the code organized by sections ?